list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The HTML is mixed with PHP logic, reducing readability and maintainability.

Recommendations: Separate PHP logic from HTML using a template engine or a simple templating approach.

Example:
```php
// PHP logic
$books = [];
while ($row = mysqli_fetch_assoc($result)) {
    $books[] = $row;
}
include 'list_books_template.php';

// list_books_template.php
foreach ($books as $book) {
    echo "<tr><td>{$book['title']}</td><td>{$book['author']}</td><td>{$book['year']}</td><td>...</td></tr>";
}
```